TrackletAlgorithm/Constants.h

```diff
@@ -19,6 +19,8 @@ constexpr int kNTEUnitsLayerDisk[] = {0, 5, 2, 5, 0, 3, 3, 3, 0, 2, 0}; // Numbe
 // The outer layerDisk of the seed type is used as index, e.g. kNTEUnits[TF::L1L2] == kNTEUnitsLayerDisk[TF::L2]. N.B.: D1 actually has two seed types (L1D1, L2D1), take the largest value for now...
 constexpr int kNbitsrzbin = 3;
 constexpr int kNbitsphibin = 3;
+constexpr unsigned kNbitsrzbinMP = kNbitsrzbin + 1;
```

Comment:
I don't understand why you need these variables that are 4. Can you explain.

Reply:
It seems the disk don't run properly with the R/Z bins set to 3 bits. I believe the VMRouterCM also uses 4.